db/migrate/20220323125356_add_terms_of_use_accepted_date_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddTermsOfUseAcceptedDateToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :terms_of_use_accepted_date, :datetime
```

Comment:
There's a convention to end date fields with _at. Maybe you rename this to terms_of_use_accepted_at to make the code more readable.

Reply:
i changed column names to _at